sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/identifiers.scala

```diff
@@ -58,6 +58,16 @@ sealed trait CatalystIdentifier {
     }
   }
 
+  def parts: Seq[String] = {
```

Comment:
`nameParts` is a more commonly used name in Spark.

Reply:
Done